### DISCUSSION TITLE
想加个星座占卜的插件

### DISCUSSION THREAD

**Discussion by huihuiismywife** [nonebot/discussions#166] [Polls]
**是否在使用中遇到某些问题而需要新的特性？请描述：**

如题，有群友想在群里做星座占卜，就过来跪求大佬们了 _(:з」∠)_

**描述你所需要的特性：**

星座占卜(或者其他什么的)+你的生日/你的星座    就可以开始占卜了
emmmmm如果可以的话，能配上一些图片吗，比如星座的图片或者其他的 _(:з」∠)_

**Comment by j1g5awi** [comment]
把你的 Issue 转移到论坛了，NoneBot2 仓库的 Issue 并不是用来求插件的，请求新特性指的是 NoneBot2 框架的新特性。

**Comment by huihuiismywife** [comment]
好叭 _(:з」∠)_ 麻烦大佬了
